config/hodgepodge.cfg

```diff
@@ -333,6 +477,7 @@ tweaks {
 
     # Makes the chat history longer instead of 100 lines
     B:longerChat=true
+    B:longerSentMessages=true
```

Comment:
this doesn't exist in the master branch of hodgepodge

Reply:
done